app/src/main/java/org/schabi/newpipe/player/PopupVideoPlayer.java

```diff
@@ -296,9 +298,7 @@ private void resetNotification() {
                 .setSmallIcon(R.drawable.ic_newpipe_triangle_white)
                 .setVisibility(NotificationCompat.VISIBILITY_PUBLIC)
                 .setContent(notRemoteView);
-        if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.JELLY_BEAN) {
```

Comment:
CC @friendlyanon

Reply:
Thank you for the mention. Luckily JB is API 16, so this is not an issue :)